### ISSUE TITLE
[1.21.1] Extradelight latest version issue loading worlds

### ISSUE THREAD

**Issue by Periam** [Lance5057/ExtraDelight#134]
After installing the latest version of the mod, the worlds (new or old) does not load.

Minecraft 1.21.1
Neoforge-21.1.73

Mods list
configuration-neoforge-1.21.1-3.1.0
extadelight-2.1.1
farmersdelight-1.21.1-1.2.4a
gtceu-1.21.1-1.4.6
jei-1.21.1-neoforge-19.21.0.247
ldlib-neoforge-1.21.1-1.0.29.b
patchouli-1.21-87-NEOFORGE

Logs
[latest.log](https://github.com/user-attachments/files/17601121/latest.log)

Screen
![image](https://github.com/user-attachments/assets/b23d732f-ab05-4529-98fa-cecab3ca77e6)

**Comment by Lance5057** [comment]
does removing extra delight let you proceed? cause this doesn't appear to be my error

**Comment by Periam** [comment]
Yes,. it does let me load worlds.

**Comment by Lance5057** [comment]
Try removing gtceu-1.21.1-1.4.6 and seeing if it launches

**Comment by Periam** [comment]
If I remove the gtceu 1.21.1-1.4.6, the game launch, I initially discarded the GTCEU mod since before to the latest Extra delight update the game was working...

**Comment by Lance5057** [comment]
looking at the log gtceu seems to be doing something that causes conflict but i dont know what they're doing.

**Comment by Periam** [comment]
Should I open a new ticket to the GTCEU team?

**Comment by Lance5057** [comment]
Yeah probably a good idea, if you wouldn't mind linking it here too

**Comment by Periam** [comment]
I will open a new one, I do not know how to link it ... sorry

**Comment by Lance5057** [comment]
its the chain link icon above, after the H B I

**Comment by Periam** [comment]
https://github.com/GregTechCEu/GregTech-Modern/issues/2282

**Comment by Lance5057** [comment]
Looks like this isn't me so I'm gonna close this.

**Comment by Periam** [comment]
hi did you check the post made recently in the  issue opened in greg tech?

**Comment by Lance5057** [comment]
Interesting, Ill look into it

**Comment by Lance5057** [comment]
Figured it out and fixed it.
